projects/python/image_augmentation.md

```diff
@@ -0,0 +1,8 @@
+# image-augmentation
+
+[![](https://img.shields.io/badge/project-link-green)](https://github.com/kurttepelikerim/image_augmentor)
```

Comment:
Bad link

Reply:
I mistakenly linked the wrong project path but the original project repo was created before the hw2 deadline ('Updated 4 days ago').